.gitignore

```diff
@@ -1 +1,2 @@
 *~
+.vscode
```

Comment:
What sort of files does vscode create?

Reply:
Thanks - first answer - yes I'll take a look. 

In the .vscode folder is a settigs.json file that holds a ...

`{
    "FSharp.suggestGitignore": false
}`

Which to me seemed odd to me.